components/central-application-connectivity-validator/cmd/centralapplicationconnectivityvalidator/centralapplicationconnectivityvalidator.go

```diff
@@ -104,8 +104,6 @@ func main() {
 		options.eventingPathPrefixEvents,
 		options.eventingPublisherHost,
 		options.eventingDestinationPath,
-		options.appRegistryPathPrefix,
```

Comment:
Be aware that removing this from the application's code is not enough. It should be removed from deployment.